config/data/RevenueLineItems.php

```diff
@@ -6,18 +6,6 @@
     ]
 ];
 
-$GLOBALS['dataTool']['RevenueLineItems']['account_id'] = [
-    'related' => [
-        'module' => 'Accounts',
-    ]
-];
-
```

Comment:
Looks like too much has been erased. RLI records should have relationships with Account module.

Reply:
Fixed